checks/config.go

```diff
@@ -57,12 +59,24 @@ type Config struct {
 	// []string{".*", "_*"}.  This is a glob that is applied to each path
 	// component of each file.
 	IgnorePatterns []string `yaml:"ignore_patterns"`
+
+	// MaxConcurrent, if not zero, is the maximum number of concurrent processes
```

Comment:
After some thought, I think command-line is appropriate. The value is going to be dependent on the underlying testing system(s). The concurrency value stored alongside the system spec in .travis.yaml tightly couples those. It also allows for the possibility of other CI systems (e.g., Chromium infra) to test the same repository without needing to share the concurrency value.

Then again, it would be the first command-line argument that needs to be passed. Maybe it should be both in config and command-line overridable? That's simple enough.